convert-apps.sh

```
@@ -579,13 +583,17 @@ get_existing_runtipi_port() {
 # Initialize port tracker with all existing ports from destination repositories
 init_port_tracker() {
     local port_track_file="$OUTPUT_DIR/.port_tracker"
+    local port_map_file="$OUTPUT_DIR/.port_map"
     
-    # Clear any existing port tracker
+    # Clear any existing port tracker files
     > "$port_track_file"
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)